app/build.gradle

```diff
@@ -97,7 +101,14 @@ dependencies {
     compile "com.android.support:recyclerview-v7:${rootProject.ext.androidSupportSdkVersion}"
     compile "com.squareup.leakcanary:leakcanary-android:1.3.1"
 
-    apt 'com.google.dagger:dagger-compiler:2.0.1'
+    apt 'com.google.dagger:dagger-compiler:2.5'
+    apt "com.ryanharter.auto.value:auto-value-gson:0.3.1"
+    apt "com.google.auto.value:auto-value:1.2"
+    // used for AndroidStudio sync
+    provided "com.google.auto.value:auto-value:1.2"
+    // to make OkBuck work with SqlDelight
+    provided 'com.github.okbuilds:SqlDelightBin:0.4.3'
```

Comment:
Can you push the sqldelightbin code under okbuck in github so others can contribute if a newer version is released and they need to use it?

I worry about this because the sqldelight plugin version might not match what users use in gradle. We should put a warning message to the output when this happens so they are aware